content/en/agent/proxy.md

```diff
@@ -412,47 +683,59 @@ If you are using older version of HAProxy, you have to reload or restart HAProxy
 Edit each Agent to point to HAProxy by setting its `dd_url` to the address of HAProxy, for example: `haproxy.example.com`.
 This `dd_url` setting can be found in the `datadog.yaml` file.
 
-`dd_url: http://haproxy.example.com:3834`
+`dd_url: <SCHEME>://haproxy.example.com:3834`
+
+Replace `<SCHEME>` by `https` if you want your data to be encrypted between the Agent and HAProxy, or by `http` otherwise.
 
 To send traces, profiles, processes, and logs through the proxy, setup the following in the `datadog.yaml` file:
 
 ```yaml
 apm_config:
-    apm_dd_url: http://haproxy.example.com:3835
-    profiling_dd_url: http://haproxy.example.com:3836
+    apm_dd_url: <SCHEME>://haproxy.example.com:3835
+    profiling_dd_url: <SCHEME>://haproxy.example.com:3836
     telemetry:
-        dd_url: http://haproxy.example.com:3843
+        dd_url: <SCHEME>://haproxy.example.com:3843
 
 process_config:
-    process_dd_url: http://haproxy.example.com:3837
+    process_dd_url: <SCHEME>://haproxy.example.com:3837
 
 logs_config:
     use_http: true
     logs_dd_url: haproxy.example.com:3838
+    # Comment the line below to use encryption between the Agent and HAProxy
     logs_no_ssl: true
 
 database_monitoring:
     metrics:
         logs_dd_url: haproxy.example.com:3839
+        # Comment the line below to use encryption between the Agent and HAProxy
         logs_no_ssl: true
     activity:
         logs_dd_url: haproxy.example.com:3839
+        # Comment the line below to use encryption between the Agent and HAProxy
         logs_no_ssl: true
     samples:
         logs_dd_url: haproxy.example.com:3840
+        # Comment the line below to use encryption between the Agent and HAProxy
         logs_no_ssl: true
 
 network_devices:
     metadata:
         logs_dd_url: haproxy.example.com:3841
+        # Comment the line below to use encryption between the Agent and HAProxy
         logs_no_ssl: true
     snmp_traps:
         forwarder:
             logs_dd_url: haproxy.example.com:3842
+            # Comment the line below to use encryption between the Agent and HAProxy
             logs_no_ssl: true
+
+appsec_config (deprecated):
+    appsec_dd_url: haproxy.example.com:3844
 ```
 
-Then edit the `datadog.yaml` Agent configuration file and set `skip_ssl_validation` to `true`. This is needed to make the Agent ignore the discrepancy between the hostname on the SSL certificate ({{< region-param key="dd_full_site" code="true" >}}) and your HAProxy hostname:
+When using encryption, if the Agent cannot access the proxy certificate or is unable to validate it, you can then edit the `datadog.yaml` Agent configuration file and set `skip_ssl_validation` to `true`.
+With this option set to `true`, the Agent will skip the certificate validation step so it will not be able to verify the identity of the proxy but the communication will still be encrypted.
```

Comment:
```suggestion
With this option set to `true`, the Agent will skip the certificate validation step so it will not verify the identity of the proxy but the communication will still be encrypted with SSL/TLS.
```